src/main/java/com/hadoop/compression/lzo/DistributedLzoIndexer.java

```diff
@@ -66,6 +76,20 @@ private void walkPath(Path path, PathFilter pathFilter, List<Path> accumulator)
     }
   }
 
+  static void setJobName(Job job, String[] args) {
+    final Configuration conf = job.getConfiguration();
+
+    String name = conf.get(JOB_NAME_KEY, "Distributed Lzo Indexer " + Arrays.toString(args));
```

Comment:
Can we define a constant for "Distributed Lzo Indexer" so that it can be reused by the test?